backend/app/services/im/manager.py

```
@@ -0,0 +1,431 @@
+
+                # Wait for AI response and send it
+                await self._wait_and_send_response(
+                    message, team_id, session.task_id
+                )
+
+        except Exception as e:
+            logger.error(f"Error creating task: {e}", exc_info=True)
+            await self._send_error_message(
+                message,
+                team_id,
+                f"Failed to process your message: {str(e)}",
```

Ruff (0.14.13)

Use explicit conversion flag

Replace with conversion flag

(RUF010)